sign/dilithium/eddilithium3.go

```diff
@@ -0,0 +1,96 @@
+// Code generated from mode.templ.go. DO NOT EDIT.
+
+package dilithium
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/cloudflare/circl/sign/dilithium/internal/common"
+	"github.com/cloudflare/circl/sign/eddilithium3"
+)
+
+// implEdMode3 implements the mode.Mode interface for EdDilithium3.
+type implEdMode3 struct{}
+
+// EdMode3 is Dilithium in mode "EdDilithium3".
+var EdMode3 Mode = &implEdMode3{}
+
+func (m *implEdMode3) GenerateKey(rand io.Reader) (PublicKey,
+	PrivateKey, error) {
+	return eddilithium3.GenerateKey(rand)
+}
+
+func (m *implEdMode3) NewKeyFromExpandedSeed(seed *[96]byte) (PublicKey,
+	PrivateKey) {
+	panic("Not supported for hybrid modes")
+}
+
+func (m *implEdMode3) NewKeyFromSeed(seed []byte) (PublicKey,
+	PrivateKey) {
+	if len(seed) != common.SeedSize {
+		panic(fmt.Sprintf("seed must be of length %d", common.SeedSize))
```

Comment:
`fmt.Errorf`?

Reply:
In other places we raise a string.  Why be inconsistent and raise an error here instead of a string?

Reply:
Ah, I see I'm mistaken: it's inconsistent already.  I'll choose one. Thanks.